### PR TITLE
updates image and load reset hover only on navigator table

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

- updates filer_2 preview image
- reset-hover class only on navigator table

### CODE REVIEW

**Comment by Flight** [comment]
LGTM

File: filer/static/filer/js/addons/table-dropzone.js [hunk 1]
```diff
@@ -68,8 +68,6 @@
                         folderName.text(folderTitle);
                     },
                     dragleave: function (e) {
```
Comment (vxsx) [inline filer/static/filer/js/addons/table-dropzone.js]: no need for e anymore

**Comment by vxsx** [comment]
:+1:

**Comment by lory87** [comment]
There are some problems on travis. @mikek is already on it. gulp ci is ok so I merge this pr

**Pull request merged**